### DISCUSSION TITLE
Adding 3xl to the defaults

### DISCUSSION THREAD

**Discussion by Nefcanto** [tailwindlabs/tailwindcss#10734] [Ideas]
Right now Tailwind only supports up to 2xl or 1536px.

Can you add another step too? Monitors are getting larger and pixel densities are going higher. Recently we noticed that 2xl is not enough for some larger monitors.

Of course we can customize it for our projects using `tailwind.config.js`. But it would be a good idea to add another step officially as a default.

**Comment by MasterScrat** [comment]
I second this idea, and would suggest adding both 3xl and 4xl to defaults. Screens with over 3000px horizontally are not so uncommon anymore.

**Comment by Harshnaruto7** [comment]
Bring 3xl, we are having problems with this, as the screens are getting bigger we are using tailwind so . we are waiting for the update.

**Comment by cgcb** [comment]
I agree!

**Comment by arthur-adriansens** [comment]
Good idea, I have a laptop screen that's 2560 wide, so xl or 2xl isn't big enough.

**Reply by zori150** [reply]
Hihi
